src/structures/Message.js

```diff
@@ -548,7 +548,11 @@ class Message extends Base {
    * @readonly
    */
   get editable() {
-    return this.author.id === this.client.user.id;
+    return (
+      this.author.id === this.client.user.id &&
```

Comment:
Shouldn't this (and `pinnable`) check if the message has been deleted too (like on `deletable`)?
```suggestion
      this.author.id === this.client.user.id && !this.deleted &&
```